src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -461,6 +480,18 @@ function CreatePipelineRun() {
             defaultMessage: 'Create PipelineRun'
           })}
         </h1>
+        <div className="tkn--create--yaml-mode">
+          <Button
+            kind="tertiary"
+            id="format-pipeline-button"
```

Comment:
nit: `id="create-pipelinerun--mode-button"`
for consistency with other ids and use of the word 'mode' already used elsewhere